tpot/export_utils.py

```diff
@@ -388,7 +348,7 @@ def replace_function_calls(pipeline_list):
 
         elif operator_name == '_multinomial_nb':
             alpha = float(operator[3])
-            fit_prior = (int(operator[4]) % 2) == 0
+            fit_prior = int(operator[4])
```

Comment:
Given what we've seen today that probably would have worked, but a good catch nonetheless.